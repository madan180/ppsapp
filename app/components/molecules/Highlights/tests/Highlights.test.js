import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Highlights from '../index';

describe('<Highlights />', () => {
  let HighlightsComponent = '';
  beforeEach(() => {
    HighlightsComponent = shallow(<Highlights>Test</Highlights>);
  });

  test('should render correctly', () => {
    expect(HighlightsComponent).toMatchSnapshot();
  });
});
