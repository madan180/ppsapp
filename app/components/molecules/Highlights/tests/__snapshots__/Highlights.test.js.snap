// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Highlights /> should render correctly 1`] = `
<Highlights
  className=""
>
  Test
</Highlights>
`;
